crates/mun_memory/src/diff.rs

```diff
@@ -7,25 +7,29 @@ use crate::{
     TypeFields,
 };
 
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
 pub enum FieldEditKind {
```

Comment:
It would be nice if you can add some comments here and in the enum below that describe what the diffs represent.